pkg/version/version_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFullVersion(t *testing.T) {
```

Comment:
1. Table driven test is more suitable for this one, instead of the other one which has only one subtest.
```
testCases := []struct {
    version string
    releaseStatus string
    ....
} {
    ...
}
for _, tc := range testCases {
    t.Run(tc.name, func(t *testing.T) {
        Version = tc.version
        ...
        assert.Equal(t, tc.expectedVersion, gotVersion)
}

```
2. The global variables should be restored to its original value after every subtest finishes, otherwise side effect would remain, which would affect other tests' results when the test runs or doesn't run.